README.md

```diff
@@ -139,12 +135,14 @@ jobs:
             -d 'grant_type=client_credentials' \
             -d 'scope=2ff814a6-3304-4ab8-85cb-cd0e6f879c1d%2F.default' \
             -d 'client_secret=${{ secrets.AZURE_SP_CLIENT_SECRET }}' |  jq -r  '.access_token')" >> $GITHUB_ENV
-      - name: Trigger notebook from PR branch
+      - name: Trigger model training notebook from PR branch
         uses: databricks/run-notebook@v0
         with:
-          local-notebook-path: notebooks/MainNotebook.py
-          # Alternatively, specify an existing-cluster-id to run against an existing cluster.
-          # The cluter JSON below is for Azure Databricks. On AWS and GCP, set
+          # Run our notebook against a remote repo
+          local-notebook-path: notebooks/deployments/MainNotebook
+          # If this is triggered from a PR use the PR's head commit, otherwise use github.sha.
```

Comment:
```suggestion
          # If the current workflow is triggered from a PR, run notebook code from the PR's head commit, otherwise use github.sha.
```